### ISSUE TITLE
copy_from_upstream copy not idempotent any more

### ISSUE THREAD

**Issue by baentsch** [open-quantum-safe/liboqs#1406]
Prior to merging #1337, running `copy_from_upstream copy` didn't change the repo (on unchanged code). This no longer is the case now: https://github.com/open-quantum-safe/liboqs/blob/main/docs/cbom.json is always updated when this command is run.
Is this intentional & necessary, @bhess ? 
This is the reason why #1405 only tests `copy_from_upstream verify` and not for an unchanged repository after `copy_from_upstream copy`. The latter test would clearly be more powerful to protect `liboqs` from incorrect upstream integrations.

**Comment by bhess** [comment]
Yes you are right, it shouldn't be updated if the repo didn't change. I'll look into it next week.
